lib/Auth/Source/Ldap.php

```diff
@@ -49,11 +60,107 @@ public function __construct(array $info, array $config)
      *
      * @param string $username  The username the user wrote.
      * @param string $password  The password the user wrote.
-     * param array $sasl_arg  Associative array of SASL options
```

Comment:
No more SASL support then?

Reply:
Good one! Of course `symfony/ldap` supports SASL, but it has to be passed in the options-array...  I see I have some work to do to make it configurable in the authsource-config.

Looking at the old code, I don't think this worked in the past.. The login-method was given an extra parameter, but it isn't being populated anywhere..

Reply:
So maybe just add some note that it's not currently supported and that it can be added when there's shown interest.

Reply:
I've already taken care of it.. Any LDAP-options can now be set in the options-array